src/plugins/new-ui/public/webcomponents/depth.html

```diff
@@ -16,9 +16,9 @@
         margin-top: -0.6vw;
       }
       #depth {
-        font-size: 1.8vw;
+        font-size: 1.35vw;
         font-weight: bold;
-        text-align: center;
+        text-align: left;
```

Comment:
Again, this was to was to prevent larger depths (3 digits) from being too large and changing the layout of the widget. Is there a better way to prevent this?